CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 ## unreleased
 * Breaking Changes
   * CardPayments
+    * Add `status` property to `CardVaultResult`
+    * Add `didAttemptThreeDSecureAuthentication` property to `CardVaultResult`
```

Comment:
We shouldn't have a "breaking changes" section on `main` as there should only be breaking changes on `next-major-version`.

Reply:
+1: also adding classes & properties is not breaking, but rather a minor version bump (semver.org)

Reply:
addressed in 35aa9df5e5749a3dd5f9ca84af3a3bab8e6da325